recipes/pheasant/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pheasant" %}
+{% set version = "2.5.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0466da0a94eb2324dab43e4583b1251891adb012a6d264aaed42c7813cbfd07e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >3.7
+    - pip
+    - setuptools
+  run:
+    - python >3.7
```

Comment:
You will need the following under `run`:

https://github.com/daizutabi/pheasant/blob/master/setup.py#L80-L86

Reply:
@synapticarbors thanks for you inputs; I tried to address them in 5a95729bf5c3744885f356c837cb498f8502d8ef

Reply:
@synapticarbors could you please review again ? I don't know to proceed from here.
Thank you.